apps/core-app/src/renderer/src/modules/box/adapter/hooks/useClipboardChannel.ts

```
@@ -93,15 +96,15 @@ export async function getLatestClipboard(): Promise<IClipboardItem | null> {
  */
 export async function applyClipboardToActiveApp(item: IClipboardItem): Promise<boolean> {
   if (!item.id) {
-    console.error('[useClipboardChannel] Cannot apply clipboard item without ID')
+    clipboardChannelLog.error('Cannot apply clipboard item without ID')
     return false
   }
 
   try {
     const result = (await applyClipboardItem(item.id)) as ClipboardActionResult | undefined
     return result?.success !== false
   } catch (error) {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)